client/tests/integration/smartcontracts/create_nft_for_every_user_smartcontract/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "create_nft_for_every_user_smartcontract"
-version = "2.0.0-pre-rc.7"
+version = "2.0.0-pre-rc.8"
```

Comment:
Looks like we left this behind
https://github.com/appetrosyan/iroha/blob/222fb00ee439a506a31ac61960e95c57cd48a8d3/client/tests/integration/smartcontracts/mint_rose/Cargo.toml#L3

Reply:
![image](https://user-images.githubusercontent.com/49983831/189817566-a11bb801-cabd-4207-846c-bc0ef2c89ded.png)

@appetrosyan

Reply:
Fixed.